### DISCUSSION TITLE
如何获取一个文本图层的具体位置和大小呢?

### DISCUSSION THREAD

**Discussion by ys821434827** [Tencent/libpag#2075] [Q&A] [answered]
想获取文本的具体位置和大小信息,但是接口中没有找到对应的功能,只有getBounds 接口 但是该方法返回的数据并不像是位置信息.

**Comment by kevingpqi123** [comment, marked as answer]
使用的是 PAGView 或 PAGPlayer 下面的 getBounds接口，位置相对于的是 PAGView 或 PAGSurface
Returns a rectangle in pixels that defines the displaying area of the specified layer, which is in the coordinate of the PAGView.

- (CGRect)getBounds:(PAGLayer *)pagLayer
Declared In
PAGView.h

**Reply by leohandsome1688** [reply]
说了半天，也没有说出解决方案~

**Reply by kevingpqi123** [reply]
API 接口已经贴在上面了，不清楚的是啥哈

**Reply by leohandsome1688** [reply]
大哥，虽然接口贴出来了，但是怎么转成自己想要的数据，其实还是一脸懵逼，这个框架用起来很吃力，文档不全，demo全部是基本的操作，想做一些复杂的编辑啥的，基本上对于大部分的使用人员来说，都是一种灾难~

**Reply by leohandsome1688** [reply]
建议弄一个交流群吧。。。大家互相交流，这个论坛并不能及时解答大家的疑惑，虽然你们也是很辛苦的在一一解答。不要说什么效率问题，解决不了大家的疑问，所谓的效率都是扯淡
